src/docs/introduction/getting-started-with-ftw-daily/index.md

```diff
@@ -153,15 +188,16 @@ integrations work.
 
 ### Sharetribe Flex client ID and client secret
 
-To use the Marketplace API, you will need a **client ID**. You can
-request access at https://www.sharetribe.com/products/flex/
+To use the Marketplace API, you will need a **client ID**. You can sign
+up for your free Flex account at
+https://www.sharetribe.com/products/flex/
 
 When you get access, you will be able to log into Flex Console and check
 the client ID.<br /> Flex Console: _Build > Applications_
 
 In addition, FTW templates use a transaction process that includes a
 privileged transition. That makes it possible to customize pricing on
-the small server that's included to the template. **Client secret** is
+the small server that's included in the template. **Client secret** is
```

Comment:
Should we replace _small_ with _Node_ or something? Or just leave it out? Just a thought.

Reply:
Replaced `small` with `Node`